docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx>=4.0
+sphinx-argparse
```

Comment:
Is this file needed now?

Reply:
No it's not. Edited.

Reply:
This file is still in latest PR.

Reply:
Oh I thought you were asking the specific extension, my bad. This file is needed to tell Read the Docs how to build docs (using Sphinx).